docker/plugins.txt

```diff
@@ -23,7 +23,6 @@ generic-webhook-trigger
 ghprb
 git-parameter
 gradle
-greenballs
```

Comment:
Why remove greenballs it is nice!

Reply:
Its not being used with new UI. Also saw below error:
```
WARNING: Illegal reflective access by hudson.plugins.greenballs.GreenBallsPlugin (file:/var/jenkins_home/plugins/greenballs/WEB-INF/lib/greenballs.jar) to field java.awt.Color.value 
WARNING: Please consider reporting this to the maintainers of hudson.plugins.greenballs.GreenBallsPlugin
at hudson.plugins.greenballs.GreenBallFilter.doFilter(GreenBallFilter.java:64)
``` 

There is related issue opened too: https://github.com/jenkinsci/greenballs-plugin/issues/17
![image](https://user-images.githubusercontent.com/61760125/235220823-538441ae-ffcd-46dc-a980-be084386eec4.png)

Reply:
Got it, thanks!